.zuul.d/project.yaml

```diff
@@ -6,6 +6,8 @@
         - ansible-runner-build-container-image-stable-2.9
         - ansible-runner-build-container-image-stable-2.10
         - ansible-runner-build-container-image-stable-2.11
+        - ansible-tox-py36
+        - ansible-tox-py37
     gate:
       jobs: *id001
     post:
```

Comment:
So we should likely delete post / periodic jobs for now. Given we need to think about it a little.

I don't think we need to provide images build of runner 2.0, but just use the devel version of all testing. These are the images that end up on quay for others to use

Reply:
Removed those two sections.